src/main/java/de/photon/anticheataddition/modules/sentinel/ParsedPluginMessageListener.java

```diff
@@ -9,7 +9,7 @@
 public interface ParsedPluginMessageListener extends PluginMessageListener
 {
     @Override
-    default void onPluginMessageReceived(@NotNull String channel, @NotNull Player player, @NotNull byte[] message)
+    default void onPluginMessageReceived(@NotNull String channel, @NotNull Player player, byte @NotNull [] message)
```

Comment:
to me, this new order looks bad

Reply:
I wondered about that, too, but that is actually the correct order IntelliJ tells you. And the annotations are also from JetBrains. So who knows.

Reply:
i'd jsut ignore intellij and use what i like better visually. having the byte and its array indicator split by `@NotNull` is not something I would keep in my code, but if you don't mind keep it like its now.